itou/analytics/models.py

```diff
@@ -65,6 +72,23 @@ class Meta:
         unique_together = ["code", "bucket"]
         indexes = [models.Index(fields=["measured_at", "code"])]
 
+    def get_value_display(self):
+        """
+        `value` column being an integer, it's impossible to store percentage values without lowering its accuracy.
+        So, store the original value multiplied by 1000 and display it divided by 100.
+        """
+        if self.code in PERCENTAGE_DATUM:
+            return self.value / 100
+        return self.value
+
+    def save(self, *args, **kwargs):
+        self.clean()
+        super().save(*args, **kwargs)
+
+    def clean(self):
+        if not isinstance(self.value, int):
```

Comment:
Un peu plus tolérant.
```suggestion
        if int(self.value) != self.value:
```

Exemple de cas en échec avec l’implémentation actuelle : `9700 / 100`.

Reply:
En corrigeant mes tests grâce à ton commentaire précédent, je me suis rendue compte qu'une ValueError était levée avec une `value` de type string.
Cela m'a permis de me rendre compte que je n'avais pas bien saisi ton commentaire. Une value de 97 est bien un entier. :thinking: 
Je reviens donc à l'idée initiale mais je ferai une PR complémentaire pour corriger si cela te semble nécessaire.

Reply:
Je ne comprends pas, si on passe un entier en format chaine de caractères, `int` transformera la valeur :
```python
>>> int("123")
123
```